src/raster/r.damflood/main.c

```diff
@@ -766,76 +769,80 @@ int main(int argc, char *argv[])
+                    // printf("The velocity vel_max is valid: %f\n", vel_max);
+                    // while (!getchar()) {
+                    // }
                 }
             }
         }
-        G_message("the number of lake cell is': %d\n", num_cell);
+        G_message("The number of lake cell is': %d\n", num_cell);
 
-        /**************************************/
-        /* timestep in funzione di V_0 e res */
-        /**************************************/
-        // timestep=0.01;
-        // timestep= ((res_ns+res_ew)/2.0)/(vel_max*50.0);
+        /*********************************************
+         * timestep as a function of V_0 and res
+         *********************************************/
+        // timestep = 0.01;
+        // timestep = ((res_ns + res_ew) / 2.0) / (vel_max * 50.0);
         //  DEVELOPEMENT
-        //*****************************************************************************
+        //*********************************************************************/
         G_message("vel on the dam break cells is %.2f, and timestep is %.2f",
                   vel_max, timestep);
     }
 
-    //*****************************************************************************
-    // Uniform drop in of lake (method=1 or method =2)
-    //*****************************************************************************
+    /***************************************************************************
+     * Uniform drop in of lake (method=1 or method=2)
+     **************************************************************************/
     if (method == 1 || method == 2) {
-        /* calcolo l'abbassamento sul lago*/
+        /* Lake depth reduction calculation */
         if (num_cell != 0) {
             fall = (volume) / (num_cell * res_ew * res_ns);
-            // printf("volume=%f, fall=%f\n",volume,fall);
-            // while(!getchar()){ }
+            // printf("volume=%f, fall=%f\n", volume, fall);
+            // while (!getchar()) {
+            // }
         }
 
         for (row = 1; row < nrows - 1; row++) {
             for (col = 1; col < ncols - 1; col++) {
                 if (m_DAMBREAK[row][col] > 0) {
-                    // ragiona se ha senso
+                    // I think it makes sense
                     m_h2[row][col] = m_h1[row][col] - fall;
                     if (m_h2[row][col] <= 0) {
                         m_h2[row][col] = 0.0;
                         if (m_h1[row][col] > 0) {
-                            // questo warning va modificato perchè vale per ogni
-                            // cella ---> bisogna metterne uno generico che
-                            // valga quando tutte le celle sono con h=0
+                            // This warning must be modified since it is valid
+                            // for every cell ---> You have to put a generic one
+                            // that is valid when all cells have h=0
```

Comment:
```suggestion
                            // This warning is modified since it is valid
                            // for every cell ---> A generic one is needed
                            // for the case when all cells have h=0
```